<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultPrintWidgetClassTableItem</class>
 <widget class="QWidget" name="ResultPrintWidgetClassTableItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>106</width>
    <height>18</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelClassNumber">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>18</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelClass1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border-bottom: 1px solid #D9D9D9;
	border-right: 1px solid #D9D9D9;
	border-top: 0px solid #D9D9D9;
	border-left: 0px solid #D9D9D9;
	margin: 0px;
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelClass2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border-bottom: 1px solid #D9D9D9;
	border-right: 1px solid #D9D9D9;
	border-top: 0px solid #D9D9D9;
	border-left: 0px solid #D9D9D9;
	margin: 0px;
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelClass3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border-bottom: 1px solid #D9D9D9;
	border-right: 1px solid #D9D9D9;
	border-top: 0px solid #D9D9D9;
	border-left: 0px solid #D9D9D9;
	margin: 0px;
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelClass4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border-bottom: 1px solid #D9D9D9;
	border-right: 1px solid #D9D9D9;
	border-top: 0px solid #D9D9D9;
	border-left: 0px solid #D9D9D9;
	margin: 0px;
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelClass5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border-bottom: 1px solid #D9D9D9;
	border-right: 1px solid #D9D9D9;
	border-top: 0px solid #D9D9D9;
	border-left: 0px solid #D9D9D9;
	margin: 0px;
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelClass6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border-bottom: 1px solid #D9D9D9;
	border-right: 1px solid #D9D9D9;
	border-top: 0px solid #D9D9D9;
	border-left: 0px solid #D9D9D9;
	margin: 0px;
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
